backend/terraform/environments/staging/tfvars.sample

```diff
@@ -13,6 +13,8 @@ identifier              = "dev-postgres"
 instance_class          = "db.t2.small"
 storage_type            = "gp2"
 multi_az                = "false"
+replica_count           = 2
```

Comment:
One of these two seems redundant. Can we remove count and use array length to generate count

Reply:
done